vars/dockerBuildArgs.groovy

```diff
@@ -56,75 +101,46 @@ String call(Map config = [:]) {
       ret_str += " --build-arg CB0=" + current_time.get(Calendar.WEEK_OF_YEAR)
     }
     if (add_repos) {
-      if (env.REPOSITORY_URL) {
-        ret_str += ' --build-arg REPO_URL=' + env.REPOSITORY_URL
-      }
-      string repo_name = null
+      String repo_name = null
       if (stage_info['target'] == 'centos7') {
-        if ((repo_type == 'local') && (env.DAOS_STACK_EL_7_LOCAL_REPO)) {
-            repo_name = env.DAOS_STACK_EL_7_LOCAL_REPO
-        } else if ((repo_type == 'stable') &&
-                   (env.DAOS_STACK_EL_7_DOCKER_STABLE_REPO)) {
-          repo_name = env.DAOS_STACK_EL_7_DOCKER_STABLE_REPO
-        } else if ((repo_tpe == 'release') &&
-                   (env.DAOS_STACK_EL_7_DOCKER_RELEASE_REPO)) {
-          repo_name = env.DAOS_STACK_EL_7_DOCKER_RELEASE_REPO
-        } else if ((repo_type == 'dev') &&
-                   (env.DAOS_STACK_EL_7_DOCKER_DEV_REPO)) {
-          repo_name = env.DAOS_STACK_EL_7_DOCKER_DEV_REPO
-        }
+        repo_alias = 'EL_7'
+        repo_name = env["DAOS_STACK_${repo_alias}${repo_mod}_${repo_type}_REPO"]
```

Comment:
```suggestion
        repo_alias = 'EL_7'
        repo_arg = 'EL7'
```

and then assign:

```
        repo_name = env["DAOS_STACK_${repo_alias}${repo_mod}_${repo_type}_REPO"]
        ret_str += ' --build-arg REPO_' + repo_arg + '=' + repo_name
```

once at the end of the big loop, DRYing out the repeated duplicate assignments.  I didn't do all of the same edits for all of the branches below as some other cut-n-pasto(s) are making it confusing.